.travis.yml

```diff
@@ -13,9 +13,20 @@ branches:
     
 matrix:
   include:
-    - env: TARGETF=netcoreapp2.1
-    - env: TARGETF=netcoreapp2.0
+    - env:
+        - TARGETF="netcoreapp2.1"
+        - BUILD_EXAMPLES="1"
+    - env:
+        - TARGETF="netcoreapp2.0"
+        - BUILD_EXAMPLES="0"
 
 script:
+  - |
+    if [ "${BUILD_EXAMPLES}" == "1" ]
+    then
+        dotnet restore examples
+        dotnet build --no-restore examples/examples.sln -c Release /p:BuildCoreOnly=True
```

Comment:
NIT: We could use environment variable instead of passing `/p:ENV_VAR=val`

Reply:
oh nice tip, didn't know that, done!